public/components/wz-menu/wz-menu-management.js

```diff
@@ -24,20 +25,76 @@ class WzMenuManagement extends Component {
     };
 
     this.managementSections = {
-      management: { id: 'management', text: 'Management' },
-      administration: { id: 'administration', text: 'Administration' },
-      ruleset: { id: 'ruleset', text: 'Ruleset' },
-      rules: { id: 'rules', text: 'Rules' },
-      decoders: { id: 'decoders', text: 'Decoders' },
-      lists: { id: 'lists', text: 'CDB lists' },
-      groups: { id: 'groups', text: 'Groups' },
-      configuration: { id: 'configuration', text: 'Configuration' },
-      statusReports: { id: 'statusReports', text: 'Status and reports' },
-      status: { id: 'status', text: 'Status' },
-      cluster: { id: 'monitoring', text: 'Cluster' },
-      logs: { id: 'logs', text: 'Logs' },
-      reporting: { id: 'reporting', text: 'Reporting' },
-      statistics: { id: 'statistics', text: 'Statistics' },
+      management: {
+        id: WAZUH_MANAGEMENT_ID.MANAGEMENT,
+        cyTestId: WAZUH_MANAGEMENT_CY_TEST_ID.MANAGEMENT,
+        text: 'Management',
```

Comment:
do we need a comma after the last element of every block?

Reply:
this is considered good practice for some JS projects, we could choose if not.